spec/models/pacbio/request_library_spec.rb

```diff
@@ -44,4 +44,12 @@
     end
   end
 
+  context 'collection?' do
+    let(:well)                { create(:pacbio_well) }
+
+    it 'will always be true' do
+      expect(well).to be_collection
+    end
+  end
```

Comment:
Should we be creating a pacbio request library here, rather than a well, and shouldn't collection? be false rather than true?

Reply:
Yes. Just copied it from the other one.